pkgs/development/python-modules/mayim/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+
+, psycopg
+, aiosqlite
+, asyncmy
+
+# test
+, pytest
+, pytest-asyncio
+
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mayim";
+  version = "1.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ahopkins";
+    repo = "mayim";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-nb0E9kMEJUihaCp8RnqGh0nSyDQo50eL1C4K5lBPlPQ=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```
Please use `postPatch` instead. Overriding patchPhase may cause unintended bugs in the future.

Besides this, LGTM.

---

Alright, that makes sense